tests/testthat/test-stats-function.r

```diff
@@ -34,3 +34,16 @@ test_that("works with discrete x", {
   expect_equal(ret$x, 1:2)
   expect_equal(ret$y, 1:2)
 })
+
+test_that("works with formula syntax", {
+  dat <- data_frame(x = 1:10)
+
+  base <- ggplot(dat, aes(x, group = 1)) +
+    stat_function(fun = ~ .x^2, geom = "point", n = 5) +
+    scale_x_continuous(limits = c(0, 10))
+  (ret <- layer_data(base))
```

Comment:
Is there a reason this assignment is in parentheses?

Reply:
No, accidental leave-in from testing. :)
```suggestion
  ret <- layer_data(base)
```